utils/githooks/README.md

```diff
@@ -8,68 +8,78 @@ The DAOS repo contains several built-in githooks that are intended
 and to avoid some common mistakes in the development cycle.
 
-To enable these standard githooks requires a two step process:
+## Install DAOS Git Hooks
+Installing is a two-step process:
 
-1. Install the hooks
-
-Configure your core.hookspath as follows (Recommended):
+### 1. Install the hooks
 
+Recommended: Configure your `core.hookspath`.  
+Any new githooks added to the repository will automatically run,
+but possibly require additional software to produce the desired effect.
+Additionally, as the branch changes, the githooks change with it.
 ```sh
 git config core.hookspath utils/githooks
 ```
 
 Additionally, one can copy the files into an already configured path.
 
-With the first option, any new githooks added to the repository will
-automatically run, but possibly require additional software to produce the
-desired effect.  Additionally, as the branch changes, the githooks
-change with it.
-
-2. Install all of the required tools
+### 2. Install the required tools
 
 The Githooks framework in DAOS is such that the hooks will all run.
 However, some hooks will simply check for required software and are
 effectively a noop if such is not installed.
 
-On many systems, the required packages can be installed through standard means
-but customization may be required.  Some are specified in
-[requirements.txt](../../requirements.txt) so can be installed using
-`pip install -r requirements.txt` and `pip install -r utils/cq/requirements.txt`
-which can also be done using a virtual environment. The following
-packages are used by built-in githooks.
+Requirements come from a combination of `pip` and system packages and can usually be installed through standard means.  
+To install `pip` packages specified in [utils/cq/requirements.txt](../../utils/cq/requirements.txt):
+```sh
+python3 -m pip install -r utils/cq/requirements.txt
+```
+To install system packages with your package manager - for example:
+```sh
+dnf install git-clang-format
+```
+### Installed tools
+The following packages are used by built-in githooks.
 
-1. clang-format version 14.0.5 or higher.  If the check is unable to parse
-the version output, it will fail.  Try running
+1. `clang-format` version 14.0.5 or higher - Formatter for C code. If the check is unable to parse
+the version output, it will fail. Try running
 `<root>/site_scons/site_tools/extra/extra.py` to check.
-2. pylint
-3. flake8
-4. yamllint
-5. gofmt
-
-There is a daos wrapper around pylint at `utils/cq/daos_pylint.py` which will perform standard
-pylint checks whilst managing scons and PYTHONPATH setup changes automatically.  Installing
-the python packages in `utils/cq/requirements.txt` will allow it to test for all the python dependencies.
+2. `pylint`
+3. `flake8`
+4. `isort`
+5. `yamllint`
+6. `gofmt`
+7. `codespell`
 
-It is important to check the output on commit for any errors that may indicate
-any one of the required tools is missing.
+### Optional tools
```

Comment:
`#### Optional tools`
Is it a subsection of **Install the~~ required~~ tools**?